master.js

```diff
@@ -96,6 +96,10 @@ database.initialize(configJS.databaseURL).catch(err => {
 		}
 
 		winston.silly("Confirming shardTotal config value.");
+		if(Number.isInteger(configJS.shardTotal) === false && configJS.shardTotal !== "auto") {
```

Comment:
Could be `if (!parseInt(configJS.shardTotal) || configJS.shardTotal !== "auto")`

Reply:
Oh right JS.

Reply:
I'll change it.